src/main/java/com/xceptance/neodymium/module/statement/browser/multibrowser/RandomBrowser.java

```diff
@@ -0,0 +1,16 @@
+package com.xceptance.neodymium.module.statement.browser.multibrowser;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(
+{
+  ElementType.TYPE, ElementType.METHOD
+})
+public @interface RandomBrowser
+{
+    int value() default 1;
```

Comment:
We should think about the edge cases and test for them in the implementation of the `BrowserStatement` class.

Add a general JavaDoc to the annotation class that explains the usage and states that every value below 1 will deactivate this annotation (similar) to `DataSet`

Implement test cases that cover this.

Reply:
Done